src/corehost/common/pal.h

```diff
@@ -221,6 +221,7 @@ namespace pal
     bool get_global_dotnet_dirs(std::vector<pal::string_t>* recv);
 
     bool get_default_installation_dir(pal::string_t* recv);
+    bool get_sdk_self_registered_dir(pal::string_t* recv);
```

Comment:
There doesn't seem to be a point in having this in the pal header. The function is not used outside of pal.
Removing it would also let us remove it from the Unix pal (where it's not used at all).